masterlist.yaml

```
@@ -2784,31 +2784,102 @@ plugins:
         name: 'Even Better Mod Descriptions'
     msg:
```

You could add a link to your mod in locations.

```suggestion
        name: 'Even Better Mod Descriptions'
      - link: 'https://www.nexusmods.com/fallout4/mods/74459/'
        name: 'Illusive Fixes and Merges'
    msg:
```